featuretools/primitives/aggregation/api.py

```diff
@@ -0,0 +1,7 @@
+from featuretools.primitives.aggregation import *
+from warnings import warn
+
+warn(
```

Comment:
What is this? Users have to change their import statements just because we re-organized directories? 
I don't think that is ideal

Reply:
Only if they happen to be importing that far down from `featuretools.aggregation.api`